src/extended_json.ts

```diff
@@ -365,9 +403,11 @@ export namespace EJSON {
       replacer = undefined;
       space = 0;
     }
-    options = Object.assign({}, { relaxed: true, legacy: false }, options);
+    const serializeOptions = Object.assign({ relaxed: true, legacy: false }, options, {
+      seenObjects: [{ propertyName: '(input)', obj: null }]
```

Comment:
```suggestion
      seenObjects: [{ propertyName: '(root)', obj: null }]
```

[nit]: Just a preference for using `(root)` here, unless there's precedence from elsewhere for using `(input)`. No need to block approval from me on this opinion, we'll see what the others say.

Reply:
Sure, that’s a good point. I like `(root)` as well.